languages/swift/iOS/App/ContentView.swift

```diff
@@ -320,6 +332,40 @@ struct ContentView: View {
             })
         print(decryptedFolders)
     }
+
+    func authenticatorTest(clientFido: ClientFido2) async throws {
+        let ui = UserInterfaceImpl()
+        let cs = CredentialStoreImpl()
+        let authenticator = clientFido.authenticator(userInterface: ui, credentialStore: cs)
+
+        // Make credential
+        try await authenticator.makeCredential(request: MakeCredentialRequest(
+            clientDataHash: Data(),
+            rp: PublicKeyCredentialRpEntity(id: "abc", name: "test"),
+            user: PublicKeyCredentialUserEntity(id: Data(), displayName: "b", name: "c"),
+            pubKeyCredParams: [PublicKeyCredentialParameters(ty: "public-key", alg: 0)],
+            excludeList: nil,
+            requireResidentKey: true,
+            extensions: nil
+        ))
+
+        // Get Assertion
+        try await authenticator.getAssertion(request: GetAssertionRequest(
+            rpId: "",
+            clientDataHash: Data(),
+            allowList: nil,
+            options: Options(rk: true, uv: Uv.preferred),
+            extensions: nil
+        ))
+        
+        try await authenticator.silentlyDiscoverCredentials(rpId: "")
+        
+        // Only on android!
+        let client = clientFido.client(userInterface: ui, credentialStore: cs)
```

Comment:
Do we want to hide this impl behind a cfg to make sure it's not usable here?

Reply:
Maybe, I think it's better to do it once we fully understand everything though.